rohmu/notifier/http.py

```diff
@@ -0,0 +1,98 @@
+"""Copyright (c) 2022 Aiven, Helsinki, Finland. https://aiven.io/"""
```

Comment:
This module is basically only doing IO. I could write tests for it, but they would basically endup testing the mock implementation instead because there is not much logic besides logging / http requests.